### PR TITLE
feat(docs): Add skeleton mdbook config and docs

### PR DESCRIPTION
**Pull request by pdehaan**
## Description

- [ ] This PR conforms to the [Contribution Guidelines](/CONTRIBUTING)
- [ ] AI tools were used in generating this pull request

## Issues

Closes: # ECTEEN-107

### CODE REVIEW

File: book.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[book]
+authors = ["Kat Anderson", "Benjamin Forehand", "Ankita", "pdehaan"]
```
Comment (Trinaa) [inline book.toml:2]:
```suggestion
authors = ["Kat Anderson (@kanderson)", "Benjamin Forehand Jr. (@b4hand)", "Ankita Shrivastava (@ashrivastava)", "Peter deHaan (@pdehaan)"]
```
or maybe more generic

```suggestion
authors = ["Ecosystem Test Engineering Team"]
```

File: docs/cli/README.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+```
+
+In order to use this command, you need to make sure you set your [personal CircleCI token](https://circleci.com/docs/managing-api-tokens/) in your local config.ini file, as seen below:
+
+```ini
+[circleci_scraper]
+token = <YoUr_tOkEn_hErE>
+```
+
+By default, only the 2 previous days of data is fetched. If you want to customize how many previous days are fetched, you can set the `days_of_data` option in your local config.ini file:
```
Comment (Trinaa) [inline docs/cli/README.md:165]:
```suggestion
If the `days_of_data` option is not present in the config.ini file, the default of "all available data" will be fetched. If you want to customize how many previous days are fetched, you can set the `days_of_data` option in your local config.ini file:
```

File: docs/contributors.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Contributors
+
+- Kat
+- Ben
+- Ankita
+- pdehaan
```
Comment (Trinaa) [inline docs/contributors.md:6]:
```suggestion
The ecosystem-test-scripts repository is owned by the Ecosystem Test Engineering Team.
```

**Review by Trinaa — APPROVED** [review]
Looks awesome Peter! I made some suggestions (take or leave) 😸

File: book.toml (hunk 1 above)
Comment (jrbenny35) [reply book.toml:2]: This reminds me I need to change my github name

File: docs/SUMMARY.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- [Command Line Tool](./cli/README.md)
+  - [`install`](./cli/install.md)
+  - [`clean`](./cli/clean.md)
+  - [`check`](./cli/check.md)
+  - [`format`](./cli/format.md)
+  - [`test`](./cli/test.md)
+  - [`test_coverage`](./cli/test_coverage.md)
+  - [`test_coverage_html`](./cli/test_coverage_html.md)
+  - [`run_circleci_scraper`](./cli/run_circleci_scraper.md)
+  - [`run_test_metric_reporter`](./cli/run_test_metric_reporter.md)
```
[the remark this answers is not in the document]
Comment (Trinaa) [reply docs/SUMMARY.md]: I can update the change in my PR so that it aligns

File: docs/contributors.md (hunk 3 above)
Comment (janbrasna) [reply docs/contributors.md:6]: Or if you feel extra fancy you can add a dynamic contributors list via an image with avatars:

```suggestion
The ecosystem-test-scripts repository is owned by the Ecosystem Test Engineering Team.

<a href="https://github.com/mozilla/ecosystem-test-scripts/graphs/contributors"><img src="https://contrib.rocks/image?repo=mozilla/ecosystem-test-scripts" alt="Contributors graph. Avatars provided by https://contrib.rocks"></a>
```

(that is currently empty, but will expand over time — see e.g. https://contrib.rocks/preview?repo=mozilla/normandy …)

**Pull request merged**